Rewrite landing navigation (Getting Started) (#1217)
prysmaticlabs/documentation: docs/learn/dev-concepts/architecture-overview.md

The execution chain stores blocks containing user-generated transactions such as token transfers and smart contracts. Rather than the originally-proposed execution shards, Ethereum now scales data availability for rollups through blob-carrying transactions (introduced by EIP-4844 in the Deneb/Dencun upgrade and scaled further by PeerDAS in the Fulu/Fusaka upgrade). Ethereum proof-of-stake consensus provides a massive improvement over proof-of-work.

> The beacon chain runs through a distributed network of nodes known as [beacon nodes](/learn/dev-concepts/prysm-beacon-node.md). Participants who want to run a beacon node and help secure the network can stake 32 `ETH` to have their [validator client](/learn/dev-concepts/prysm-validator-client.md) join the overall pool of validators, whom have the responsibility of [proposing](/terminology#proposal-propose-) and [attesting](/terminology#attestation-attest-) to new blocks on the beacon chain. This deposit does not come out of nowhere however; validators transfer Ether from the [execution](/terminology#execution-layer) chain to the system through the use of a [validator deposit contract](/learn/dev-concepts/validator-deposit-contract.md).

Reviewer: ```suggestion
The beacon chain runs through a distributed network of nodes known as [beacon nodes](http://learn/dev-concepts/prysm-beacon-node.md). Participants who want to run a beacon node and help secure the network can stake 32 `ETH` to have their [validator client](http://learn/dev-concepts/prysm-validator-client.md) join the pool of validators, who are responsible for [proposing](http://terminology/#proposal-propose-) and [attesting](http://terminology/#attestation-attest-) to new blocks on the beacon chain. This deposit does not come out of nowhere; validators transfer Ether from the [execution](http://terminology/#execution-layer) chain to the system using a [validator deposit contract](http://learn/dev-concepts/validator-deposit-contract.md).
```
